.circleci/config.yml

```diff
@@ -3981,7 +3974,12 @@ workflows:
             - pre-build-cli
             - pre-build-go-binaries
       - build-operator:
-          <<: *runOnAllTagsWithPushCtx
+          <<: *runOnAllTags
+          context:
+            - quay-stackrox-io-readwrite
+            - quay-rhacs-eng-readwrite
+            - quay-rhacs-eng-readonly
+            - docker-io-push
```

Comment:
This can be removed now after https://github.com/stackrox/stackrox/pull/2352 was merged.
```suggestion
```

Reply:
Sorry, build-operator failed. I forgot to say that you should rebase from master.